package.json

```diff
@@ -11,76 +16,72 @@
   },
   "scripts": {
-    "build": "npm run clean && npm run bundle && npm run minify",
-    "bundle": "webpack --config webpack.config.js",
-    "clean": "rm -rf dist && rm -rf coverage && rm -rf s3",
-    "minify": "uglifyjs --screw-ie8 --compress warnings=false --mangle --comments --output ./dist/kinvey-angular2-sdk.min.js -- ./dist/kinvey-angular2-sdk.js ",
-    "cover": "istanbul cover _mocha -- --compilers js:babel-core/register -r babel-polyfill -s 100 --recursive test/unit/setup test/unit",
-    "docs": "esdoc -c esdoc.json",
-    "lint": "npm run lint:src",
-    "lint:src": "eslint src/**",
-    "lint:test": "eslint test/unit/**",
-    "preversion": "rm -rf node_modules && npm install && npm test",
-    "postversion": "git push && git push --tags && rm -rf dist && rm -rf coverage && rm -rf s3",
-    "s3": "npm run build && shjs ./scripts/s3.js",
-    "test": "mocha --compilers js:babel-core/register -r babel-polyfill -s 100 --recursive test/unit/setup test/unit",
-    "test:watch": "mocha -w --compilers js:babel-core/register -r babel-polyfill -s 100 --recursive test/unit/setup test/unit",
-    "transpile": "babel src --out-dir dist"
-  },
-  "dependencies": {
-    "kinvey-html5-sdk": "3.8.1"
+    "prebuild": "rimraf dist",
+    "build": "npm run build:umd && node scripts/copy-package.js && node scripts/copy-definitions.js && node scripts/copy-readme.js",
+    "build:umd": "webpack --config webpack-umd.config.ts",
+    "build:umd:watch": "npm run build:umd -- --watch",
+    "prebuild:watch": "rimraf dist",
+    "build:watch": "concurrently --raw \"npm run build:umd:watch\"",
+    "ci": "npm run lint && npm run test && npm run build",
+    "clean:all": "npm run clean && rimraf node_modules",
+    "clean": "rimraf dist",
+    "explorer": "source-map-explorer ./dist/kinvey-angular2-sdk.umd.js",
+    "lint": "npm run tslint 'src/**/*.ts'",
+    "tslint": "tslint --exclude src/kinvey.d.ts",
+    "postversion": "git push && git push --tags"
   },
+  "dependencies": {},
   "peerDependencies": {
-    "kinvey-html5-sdk": "3.8.1"
+    "@angular/common": "^4.0.0",
+    "@angular/core": "^4.0.0"
   },
   "devDependencies": {
-    "babel-cli": "6.23.0",
-    "babel-core": "6.23.1",
-    "babel-eslint": "7.1.1",
-    "babel-loader": "6.3.2",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-inline-dotenv": "^1.1.1",
-    "babel-plugin-module-resolver": "2.5.0",
-    "babel-plugin-transform-class-properties": "^6.19.0",
-    "babel-plugin-transform-flow-strip-types": "^6.21.0",
-    "babel-polyfill": "6.23.0",
-    "babel-preset-es2015": "6.22.0",
-    "babel-preset-stage-2": "6.22.0",
-    "babel-register": "6.23.0",
-    "browserstack-local": "^1.2.0",
-    "eslint": "^3.5.0",
-    "eslint-config-airbnb-base": "11.1.0",
-    "eslint-import-resolver-babel-module": "3.0.0",
-    "eslint-plugin-import": "2.2.0",
-    "expect": "^1.20.2",
-    "fs-extra": "2.0.0",
-    "istanbul": "1.1.0-alpha.1",
+    "@angular/common": "^4.0.0",
```

Comment:
Why do we need Angular as a dependency? Does it have something to do with the content replacement plugin I ask about in another comment?